src/Model/batchprocessing/BatchProcessKaplanMeier.py

```diff
@@ -0,0 +1,163 @@
+import kaplanmeier as km
+import matplotlib as mp1
+mp1.use("TkAgg")
+from matplotlib import pyplot as plt
+import pandas as pd
+from src.Model.batchprocessing.BatchProcess import BatchProcess
+from src.Model.PatientDictContainer import PatientDictContainer
+import logging
+
+
+class BatchProcessKaplanMeier(BatchProcess):
+    """
+    This class handles batch processing for the Clinical Data 2 CSV
+    process. Inherits from the BatchProcessing class.
+    """
+    # Allowed classes for ClinicalDataSR2CSV
+    allowed_classes = {
+        # Comprehensive SR
+        "1.2.840.10008.5.1.4.1.1.88.33": {
```

Comment:
theres a generic file for these ID's would be good to chuck it in there. I cant remember where it is but shouldn't be hard to find